portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,5 +42,35 @@ private String convertToJsonUsingGson(ArrayList storeCommentList) {
     String json = gson.toJson(storeCommentList);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the input from the form.
+    String fName = request.getParameter("firstname");
```

Comment:
Consider fully spelling these out over abbreviation. firstName instead of 'fname'. This will make it much easier to search for this logic in the future / for other folks to follow the usage of these variables.

Reply:
Okay Ms. Anna, I will make those changes.